### ISSUE TITLE
Profile format

### ISSUE THREAD

**Issue by pen-and-paper** [inkron/old#8]
What's the profile format going to be like? I was thinking something along the lines of Scratch profiles only more customizable, with more features available, but I'm not really sure. Also: Is there going to be a "friend" or "follow" system to easily find people you chat with?

**Comment by haylinmoore** [comment]
follow would make a sense. I'm thinking friend

**Comment by encloinc** [comment]
Every user has a unique tag, kind of like discord?

**Comment by haylinmoore** [comment]
There name though not some random junk like on discord. So everyone had a unique name

**Comment by encloinc** [comment]
Ye, also we need a design language

**Comment by haylinmoore** [comment]
design language? what is that?

**Comment by encloinc** [comment]
yes like

**Comment by encloinc** [comment]
Uniform design across the webapp

**Comment by haylinmoore** [comment]
ok

**Comment by encloinc** [comment]
We should have a communication platform while working on this to keep eachother on track

**Comment by encloinc** [comment]
have you ever used slack?

**Comment by haylinmoore** [comment]
How about gitter? We all have it

**Comment by encloinc** [comment]
Slack has a lot of team features and its really easy to use

**Comment by encloinc** [comment]
I find gitter kind of slow at times

**Comment by haylinmoore** [comment]
Fine

**Comment by encloinc** [comment]
Another option is discord but i personally find slack more productive

**Comment by haylinmoore** [comment]
I agree.

**Comment by haylinmoore** [comment]
Pls give me repo owner so I can setup gh-pages

**Comment by encloinc** [comment]
Alright

**Comment by encloinc** [comment]
What do you think about the logo btw

**Comment by haylinmoore** [comment]
It's nice. Do you have the SVG version? as I had to use a sh!tty online converter to get an SVG version

**Comment by encloinc** [comment]
Definitely!

**Comment by haylinmoore** [comment]
Please add it to the repo ASAP

**Comment by encloinc** [comment]
Ok

**Comment by encloinc** [comment]
Im going to setup the slack team

**Comment by haylinmoore** [comment]
kk

**Comment by haylinmoore** [comment]
https://inkron.github.io/

**Comment by encloinc** [comment]
Awesome. I've sent an invitation link to the email in your github profile

**Comment by pen-and-paper** [comment]
I agree with you on the "friend" thing, followers would probably make more sense in something like Scratch

**Comment by encloinc** [comment]
Pen-and-paper can i have your email to invite you to the slack team?

**Comment by encloinc** [comment]
Thankyou! Youve been invited
